addons/account/models/account_tax.py

```diff
@@ -342,9 +342,9 @@ def _compute_amount(self, base_amount, price_unit, quantity=1.0, product=None, p
             # When the price unit is equal to 0, the sign of the quantity is absorbed in base_amount then
             # a "else" case is needed.
             if base_amount:
-                return math.copysign(quantity, base_amount) * self.amount
+                return math.copysign(quantity, base_amount) * self.amount / self.env.context.get('force_fixed_rate', 1)
```

Comment:
Since the fixed tax are expressed in the company currency, I need to convert the amount in the line's currency.
It's a bit like for `force_sign`, when the amount is 0 we still need to know the sign to apply the fixed taxes in the right direction.

Reply:
I meant: this deserves to be added in the docstring